server/ladders-local.ts

```diff
@@ -90,15 +89,17 @@ class LadderStore {
 		if (this.saving) return;
 		this.saving = true;
 		const ladder = await this.getLadder();
+		if (!ladder) return;
```

Comment:
What's with all of this stuff? `getLadder` should never return a falsy value.

Reply:
Oh, that _was_ because when I was adding return values for every function, I think I added `null` for `getLadder()`, so TypeScript was complaining that it could be null, but now that I don't add return values unless they're _not_ deducible, those checks are useless.